iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -385,89 +506,69 @@ TEST_F(PortManager_test, ConditionVariablesOverflow)
                 errorHandlerCalled = true;
             });
 
-        auto conditionVariableDataResult = m_portManager->acquireConditionVariableData("AnotherToad");
-        EXPECT_TRUE(conditionVariableDataResult.has_error());
+        auto conditionVariablePort = m_portManager->acquireConditionVariableData("AnotherToad");
```

Comment:
this is also not a port and our convention for a variable which takes a `cxx::expected` is to prepend `Result`, so this would become `conditionVariableResult`